src/types.ts

```diff
@@ -1,6 +1,6 @@
+type QuizStatusType = "notStarted" | "inProgress" | "completed";
 type DifficultyType = "easy" | "medium" | "hard";
 type QuestionType = "multiple" | "boolean";
```

Comment:
ditto

Reply:
I agree with the comments for QuizStatus and Difficulty, but maybe I should keep the QuestionType as it is since I've got multiple other variables that include the word question. What do you think?

Reply:
sound good